### DISCUSSION TITLE
Model is not loading

### DISCUSSION THREAD

**Discussion by noobbuddy** [oobabooga/text-generation-webui#2172] [Q&A]
I directly download  model from Hugging Face and copy pasted in models folder and now is showing this error.

```
INFO:Loading TheBloke_wizard-mega-13B-GPTQ...
ERROR:The model could not be loaded because its type could not be inferred from its name.
ERROR:Please specify the type manually using the --model_type argument.
INFO:Loaded the model in 0.06 seconds.

INFO:Loading the extension "gallery"...
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
```

Is there solution for this other than redownloading

**Comment by Bondrake** [comment]
I think following the steps for configuring the model in the WebUI listed here will solve that: https://huggingface.co/TheBloke/wizard-mega-13B-GPTQ/blob/main/README.md

**Reply by TomLucidor** [reply]
Similar problems but with downloading and installing Mistral derivatives with broken metadata (sorry for asking but are there more general solutions?) https://huggingface.co/Undi95/Mistral-7B-claude-chat-GGUF
